jsapp/js/theme/kobo/Button.ts

```diff
@@ -0,0 +1,58 @@
+import type {ButtonVariant} from '@mantine/core';
+import {Button, rem} from '@mantine/core';
+import classes from './Button.module.css';
+
+type ButtonVariantCustom = Extract<ButtonVariant, 'filled' | 'light' | 'transparent'> | 'danger' | 'danger-secondary';
+
+declare module '@mantine/core' {
+  export interface ButtonProps {
+    variant?: ButtonVariantCustom;
+    // size?: ... // Can override ButtonFactory subset of ButtonProps only. TODO: Figure out how to forbid unused sizes.
```

Comment:
LGTM. I noticed there are two TODO comments in this file that I'm not sure if you want to iron out now or leave for the future.